UI/blog.html

```diff
@@ -43,31 +43,31 @@ <h3><a href="index.html">mr chris</a></h3>
       <h3>BEACH VACATION</h3><br/>
```

Comment:
@Mucyo-Aime-Christian good job, however, I have got some feedback on your PR. Never hard code something that you can style. for example, use CSS to put this heading in Uppercase.